src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -7,14 +7,14 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Filters out reads with more than a threshold number of N's
+ * Filters out reads with more than a threshold number of unknown bases (N).
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
 public final class AmbiguousBaseReadFilter extends ReadFilter {
 
     private static final long serialVersionUID = 1L;
 
-    @Argument(doc = "Threshold fraction of ambiguous bases",
+    @Argument(doc = "Threshold fraction of ambiguous bases.",
```

Comment:
I'd rather not add periods to argument doc strings -- they're not (or shouldn't be) sentences